AccountingSystem/AccountingSystem/GasStation.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+#include <string>
+#include "IDataHandler.h"
+
+class GasStation
+{
+public:
+    struct GasStationData
+    {
+        int ID;
+        std::string Name;
+        std::string Surname;
+        int Liters;
+        std::string Date;
+        int Ballance;
+        static int GetID(std::string line);
+    };
+
+    struct InputData
```

Comment:
We should probably pass Date as an input as well. A user might want to add row from yesterday or update the old one